monkey/monkey_island/cc/ui/src/components/report-components/zerotrust/scoutsuite/ScoutSuiteRuleButton.js

```diff
@@ -40,11 +40,11 @@ export default class ScoutSuiteRuleButton extends Component {
 }
 
 function RuleCountBadge(props) {
-  const maxRuleCountToShow = 9;
-  const textForMoreThanMaxRuleCount = maxRuleCountToShow + '+';
+  const MAX_RULE_COUNT_TO_SHOW = 9;
```

Comment:
These consts should move outside the function

Reply:
Solved by refactoring into a separate component